.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     - name: Set up Ruby
```

Comment:
Please, update this config to be similar to https://github.com/mmcs-ruby/primitives/blob/master/.github/workflows/main.yml

Code Climate Test Reporter ID has been setup in this repo.

Reply:
Done in 2a49b67